.gitignore

```diff
@@ -13,7 +13,6 @@
 
 # misc
 .DS_Store
-.env

```

Comment:
Secret keys should not be pushed on Github. While deploying the site, you can add this directly over there in environment variables.

Reply:
Done the changes